.github/workflows/testing.yml

```diff
@@ -0,0 +1,56 @@
+name: Testing
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  pytest:
+    name: Pytest (${{ matrix.os }} Python${{ matrix.python_version }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python_version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        filter: tree:0
+    - name: Set up conda environment
+      uses: conda-incubator/setup-miniconda@v3
+      with:
+        activate-environment: ci-env
+        python-version: ${{ matrix.python_version }}
+        environment-file: .github/conda_environments/docs_env.yml
```

Comment:
@ehpor  Is this one supposed to use testing_env instead?

Reply:
It should. Fixed in next commit.